docs/configuration.asciidoc

```diff
@@ -383,6 +385,18 @@ Use this option to disable any of these.
 
 Get an array of enabled instrumentations with `ElasticAPM.agent.config.enabled_instrumentations`.
 
+[float]
+[[config-enabled]]
+==== `enabled`
+[options="header"]
```

Comment:
Adds a `dynamic` badge like we've done for the other settings: https://github.com/elastic/apm-agent-ruby/pull/760

```suggestion
[options="header"]

<<dynamic-configuration, image:./images/dynamic-config.svg[] >>

```

Reply:
No, only `recording` is dynamic.
It says [here](https://github.com/elastic/apm/issues/92#issuecomment-519752096) about the `enabled` configuration:
```
This can only be set locally to the agent (e.g. in code, config file, environment variable), and is interpreted once at agent initialisation time; once disabled, disabled for the process's lifetime.
```